modules/monitoring-configuring-alert-receivers.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+[id="configuring-alert-receivers_{context}"]
+= Configuring alert receivers
+
+You can configure alert receivers to ensure that you learn about important issues with your cluster.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Within the Administrator perspective, navigate to *Administration* -> *Cluster Settings* -> *Global Configuration* -> *Alertmanager*.
++
+[NOTE]
+====
+Alternatively you can navigate to the same page through the notification draw. Select the bell icon at the top right of the {product-title} web console and choose *Configure* in the *AlertmanagerReceiverNotConfigured* alert.
+====
+
+. Select *Create Receiver* in the *Receivers* section of the page.
+
+. In the *Create Receiver* form, add a *Receiver Name* and choose a *Receiver Type* from the drop-down menu.
+
+. Edit the receiver configuration:
++
+* For PagerDuty receivers:
++
+.. Choose an integration type and add a PagerDuty integration key.
++
+.. Add the URL of your PagerDuty installation.
++
+.. Select *Show advanced configuration* if you want to edit the client and incident details or the severity specification.
++
+* For webhook receivers:
++
+.. Add the endpoint to send HTTP POST requests to.
++
+.. Select *Show advanced configuration* if you want to edit the default option to send resolved alerts to the receiver.
++
+* For email receivers:
++
+.. Add the email address to send notifications to.
++
+.. Add SMTP configuration details, including the address to send notifications from, the smarthost and port number used for sending emails, the SMTP server's hostname and authentication details.
++
+.. Choose whether TLS is required.
++
+.. Select *Show advanced configuration* if you want to edit the default option not to send resolved alerts to the receiver or edit the body of email notifications configuration.
++
+* For Slack receivers:
++
+.. Add the URL of the Slack webhook.
++
+.. Add the Slack channel or user name to send notifications to.
++
+.. Select *Show advanced configuration* if you want to edit the default option not to send resolved alerts to the receiver or edit the icon and username configuration. You can also choose whether to find and link channel names and usernames.
+
+. By default, firing alerts with labels that match all of the selectors will be sent to the receiver. If you want label values for firing alerts to be matched exactly before they are sent to the receiver:
+.. Add routing label names and values in the *Routing Labels* section of the form.
++
+.. Select *Regular Expression* if want to to use a regular expression.
++
+.. Select *Add* to add further routing labels.
```

Comment:
Select `Add Label` to add further routing labels.

Reply:
Good catch, thanks! I have now resolved this.